README.md

```diff
@@ -914,7 +914,9 @@ Server: spray-can/1.3.3
 ## GET /workflows/:version/:id/logs/:call
 
 This will return paths to the standard out and standard error files that were generated during the execution
-of a particular fully-qualified name for a call.
+of a particular fully-qualified name for a call. 
+A call has 1 or more standard out and standard err logs, depending on if the call was scattered or not.
+In the latter case, one log is provided for each instance of the call that has been run.
```

Comment:
Maybe it's worth noting that in the "scatter" case, the logs are in shard-order?  Maybe that's obvious?